src/Servers/Kestrel/test/FunctionalTests/RequestTests.cs

```diff
@@ -675,7 +675,7 @@ public async Task RequestsCanBeAbortedMidRead(ListenOptions listenOptions)
                 }
             }
 
-            await Assert.ThrowsAsync<TaskCanceledException>(async () => await readTcs.Task).DefaultTimeout();
```

Comment:
@halter73 can you confirm if this should change in master?

Reply:
Already did. He says it's no big deal. Ideally this should change in both release/2.2 and master. It can be done later if necessary.